deploy/deduplication-db-migrations/chart/templates/migrations-job.yaml

```diff
@@ -30,6 +30,6 @@ spec:
         - migrations
         - apply
         - --url
-        - "postgresql://{{ .Values.global.postgresql.auth.username }}:{{ .Values.global.postgresql.auth.password }}@{{ .Values.global.postgresql.host }}/{{ .Values.global.postgresql.auth.database }}?{{ .Values.global.postgresql.auth.querystringargs}}"
-      serviceAccountName: {{ include "enrichment_db_migrations.fullname" . }}
+        - "postgresql://{{ .Values.database.auth.username }}:{{ .Values.database.auth.password }}@{{ .Values.database.host }}/{{ .Values.database.auth.database }}?{{ .Values.database.auth.querystringargs}}"
+      serviceAccountName: {{ include "deduplication_db_migrations.fullname" . }}
 {{ end }}
```

Comment:
no empty new line at end of file as well, there is a setting in vscode if u want to enable and forget about this: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files